HW02_A_Graph-Fails.md

```diff
@@ -0,0 +1,127 @@
+What went wrong?
```

Comment:
Ok something weird happened with these files, it happened for me too, they only half saved. So the md file is showing his old bad graphs instead of your new one. but the code is correct???